glide.yaml

```diff
@@ -32,12 +32,13 @@ import:
   - lib/logutils
 - package: github.com/vishvananda/netlink
 - package: k8s.io/client-go
+  version: 82aa063804cf055e16e8911250f888bc216e8b61
   subpackages:
   - kubernetes
   - tools/clientcmd
 - package: github.com/mcuadros/go-version
 - package: github.com/projectcalico/felix
-  version: 518dbc352494a962e46d50f8bfe72a70c86951bf
+  version: 357cdd9e7fc181f56c6c197201aa39a6b0ba83c2
```

Comment:
curious, any reason we had to update this?

Reply:
I pulled the client-go version from felix, so this change was more for completeness rather than any particular feature/fix that I wanted to pull in.